packages/lib-classifier/src/plugins/drawingTools/models/tools/TemporalPointTool/TemporalPointTool.js

```diff
@@ -0,0 +1,24 @@
+import { types } from 'mobx-state-tree'
+import PointTool from '../PointTool'
+import { TemporalPoint } from '../../marks'
+
+const TemporalPointTool = types
```

Comment:
You probably already have this planned, but a note to add a `TemporalPointTool.spec` test file 😃

Reply:
I should have mentioned this in the PR - apologies. All unit tests will be in a future PR.